Runtime/Scripts/BaseMeshSync.Textures.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+namespace Unity.MeshSync {
+    partial class BaseMeshSync {
+        static readonly string[] textureNames = {
+        _MetallicGlossMap,
+        _BaseMap,
+        _MaskMap
+    };
```

Comment:
Can we move these out into a separate class, it's kind of weird seeing `BaseMeshSync._MaskMap` etc. They should probably exist in their own TextureConstants class or something

Reply:
I've moved them to MeshSyncConstants.